src/main/java/com/codicesoftware/plugins/hudson/PlasticTool.java

```diff
@@ -91,19 +91,23 @@ private String getCliLine(String[] args) {
         return builder.toString();
     }
 
-    private Reader tryExecute(String[] cmdArgs, FilePath executionPath) throws IOException, InterruptedException {
+    private Reader tryExecute(String[] cmdArgs, FilePath executionPath, boolean printOutput)
+            throws IOException, InterruptedException {
         if (executionPath == null) {
             executionPath = workspace;
         }
         ByteArrayOutputStream consoleStream = new ByteArrayOutputStream();
         Proc proc = launcher.launch().cmds(cmdArgs)
-                .stdout(new ForkOutputStream(consoleStream, listener.getLogger()))
+                .stdout(printOutput ? new ForkOutputStream(consoleStream, listener.getLogger()) : consoleStream)
```

Comment:
Following PR #36 and #37 , the changeset log command doesn't bloat the build output anymore. Does this change make sense now? Since all XML-based commands (find & log) output to a file, I'm not sure if we'll need this...

Reply:
Like mentioned in the top comment, this is something different. PR #36 and it's fix in #37 seems to be for improving memory usage. The Jenkins console output will still be cluttered with XML text. This improvement allows to execute commands in quiet mode.

Reply:
Hi,

It's not accurate to say that the Jenkins console output will still be cluttered with XML text. Commands `cm log` or `cm find` won't print anything to standard output if you tell them to write the XML output to a file.

We'll accept these changes and we'll evaluate their performance in the future to study their redundancy.